tests/large/test_ide.py

```diff
@@ -65,37 +64,19 @@ def test_default_eclipse_ide_install(self):
         proc = subprocess.Popen(self.command_as_list(self.exec_path), stdout=subprocess.DEVNULL,
                                 stderr=subprocess.DEVNULL)
 
-        # on 64 bits, there is a java subprocess, we kill that one with SIGKILL (eclipse isn't reliable on SIGTERM)
-        if self.arch_option == "x86_64":
-            self.check_and_kill_process(["java", self.arch_option, self.installed_path],
-                                        wait_before=self.TIMEOUT_START, send_sigkill=True)
-        else:
-            self.check_and_kill_process([self.exec_path],
-                                        wait_before=self.TIMEOUT_START, send_sigkill=True)
+        self.check_and_kill_process(["java", self.installed_path], wait_before=self.TIMEOUT_START)
```

Comment:
Mind expanding a bit? Why is the 32/64 bits isn't different anymore? Did you try it on 32 bits?